indexd/index/drivers/alchemy.py

```diff
@@ -1149,6 +1149,42 @@ def add_version(
 
             return record.did, record.baseid, record.rev
 
+    def add_blank_version(self, current_did, file_name=None, uploader=None):
+        """
+        Add a blank record version given did.
+        Authn/authz fields carry over from previous version.
+        """
+        with self.session as session:
+            query = session.query(IndexRecord).filter_by(did=current_did)
+
+            try:
+                old_record = query.one()
+            except NoResultFound:
+                raise NoRecordFound("no record found")
+            except MultipleResultsFound:
+                raise MultipleRecordsFound("multiple records found")
+
+            auth.authorize("update", [u.resource for u in old_record.authz])
+
+            new_record = IndexRecord()
+            did = str(uuid.uuid4())
+            if self.config.get("PREPEND_PREFIX"):
```

Comment:
`add_version` for regular non-blank records is missing this `PREPEND_PREFIX` block :(

Reply:
Yeah 😬I could add it to add_version but I didn't want to break anything.

Reply:
I think I'll add the prepend_prefix code to add_version and document it as a bugfix in this PR. What do you think?

Reply:
hey, i talked with other arcadians and i have a fix a for it, so don't worry about it
thanks :D

Reply:
Incredible, thank u!